swifttool/manager.py

```diff
@@ -0,0 +1,216 @@
+import copy
+import logging
+import math
+import os
+import shutil
+import stat
+import subprocess
+import sys
+import tempfile
+import time
+import yaml
+from logging.handlers import SysLogHandler
+from utils import get_devices, RING_TYPES, env
+from swift.common.ring import RingBuilder
+
+
+class CapacityManager(object):
```

Comment:
Need some additional information on how this will be used.  It does not appear to be referenced anywhere.